demo/demo/tests/test_migration_with_deduplication.py

```diff
@@ -163,16 +161,20 @@ def pre_migration_run(pre_apps) -> None:
                 **kwargs
             )
 
-    with pytest.raises(AssertionError):
-        run_migration(
-            'subscriptions',
-            '0026_alter_subscriptionpayment_status_and_more',
-            '0027_auto_20221109_1525',
-            pre_migration_run,
-        )
+    apps = run_migration(
+        'subscriptions',
+        '0026_alter_subscriptionpayment_status_and_more',
+        '0027_auto_20221109_1525',
+        pre_migration_run,
+    )
+
+    # Should be left untouched.
+    model = apps.get_model('subscriptions', 'SubscriptionPayment')
+    all_transaction_ids = [entry.provider_transaction_id for entry in model.objects.all()]
```

Comment:
Doesn't matter in tests, but in general you should not create model instances if you need only few columns:
```python
all_transaction_ids = model.objects.values_list('provider_transaction_id', flat=True)
```